datamade/census_area
# Reorganize code and add documentation

## Description

This PR reorganizes the code, placing the central `Census` object in `__init__.py` and moving everything else into `core.py`. It also moves some methods around in order to take advantage of documentation inheritance, so we only ever have to update docs on the parent class, even if its method is inherited by many children. Finally, it migrates Usage docs from the README to Sphinx documentation (fixing the typos identified in #11), and uses autodoc to generate API documentation for each of the Census clients available for use through this library.

## Notes

Once this comes in, note to self to change the main branch on RTD to master. We should also cut a new release to PyPi (though the updates here should not change functionality).

### Testing instructions

* View the new documentation and confirm it looks good: https://census-area.readthedocs.io/en/latest/
* Use the refactored code and confirm it works as expected.
